pkg/apis/app/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,17 +1,3 @@
-// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
do you know why was it removed?

Reply:
It's auto-generated. Some times it removes, next commit will add it again.